router-tests/websocket_test.go

```diff
@@ -2159,3 +2166,236 @@ func expectConnectAndReadCurrentTime(t *testing.T, xEnv *testenv.Environment) {
 		require.Fail(t, "expected net.Error")
 	}
 }
+
+func TestWebSocketPingIntervalForGraphQLTransportWS(t *testing.T) {
+	t.Parallel()
+
+	t.Run("epoll", func(t *testing.T) {
+
+		t.Parallel()
+
+		totalUpdates := 5
+
+		// Channel to collect pings from the router
+		pingReceived := make(chan bool, totalUpdates)
```

Comment:
What's the motivation of buffering this channel if we just skip over the blocking send later?